zerver/openapi/zulip.yaml

```diff
@@ -6042,7 +6042,7 @@ paths:
                         type: integer
                         deprecated: true
                         description: |
-                          Present if `message` is present in `fetch_event_types`.
+                          **Deprecated**; Present if `message` is present in `fetch_event_types`.
```

Comment:
I merged a3e1b9161ce184272a564cd37f8a8637939b97f5 as a more detailed rewriting of this description.